sdk/storage/storage-blob/package.json

```diff
@@ -71,6 +71,8 @@
     "build:browserzip": "gulp zip",
     "build:es6": "tsc -p tsconfig.json",
     "build:nodebrowser": "rollup -c 2>&1",
+    "build:js-samples": "npm run clean && npm run build:es6 && npm run build:nodebrowser",
```

Comment:
For debugging samples, "build:js-samples" & "build:ts-samples" are being used as prelaunch tasks in VSCode Debug config.
Is the naming fine ?